applications/matter_bridge/overlay-bt_max_connections_app_thread_nrf5340.conf

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Set 10 BLE connections, as it is an upper limit supported caused by using thread and BLE.
+CONFIG_BT_MAX_CONN=10
+
+# Set buffer sizes in a consistent way with the ones used by the network core.
+#id if oncomment it there is an error that rx buffer is out of range (minimal value is 69)
+CONFIG_BT_BUF_ACL_RX_SIZE=69
+CONFIG_BT_BUF_ACL_TX_SIZE=69
+
+# Set MTU size to fit in the single buffer and avoid fragmentation (BUF_SIZE = MTU_SIZE + 4 B of L2CAP header).
+CONFIG_BT_L2CAP_TX_MTU=65
+
+# Set max number of bridged BLE devices, which is CONFIG_BT_MAX_CONN-1, as 1 connection is reserved for Matter.
+CONFIG_BRIDGE_MAX_BRIDGED_DEVICES_NUMBER=9
+CONFIG_BT_MAX_PAIRED=9
+
+# Assume that every bridged device uses only 1 endpoint, however it can be increased if specific use case requires it.
+CONFIG_BRIDGE_MAX_DYNAMIC_ENDPOINTS_NUMBER=9
+
+CONFIG_NRF_802154_RX_BUFFERS=14
```

Comment:
Can we add a small description here, why do we reduce RX buffers for radio? I suspect that it's due to a lack of free RAM, but we'll forget about it shortly :)